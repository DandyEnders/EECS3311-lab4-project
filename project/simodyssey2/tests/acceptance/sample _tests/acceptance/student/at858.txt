move(E)
pass
move(SW)
move(N)
move(N)
pass
test(21,9,21,20,35)
move(NE)
move(SW)
move(NE)
move(E)
move(E)
move(NE)
move(SE)
wormhole
move(N)
land
test(15,24,15,28,36)
move(SW)
move(E)
pass
move(N)
wormhole
move(NW)
pass
move(SE)
move(S)
move(SW)
land
move(SW)
move(S)
move(E)
abort
move(E)
move(E)
liftoff
wormhole
move(NE)
move(SW)
move(NE)
move(NW)
move(W)
move(N)
pass
move(S)
move(SE)
move(SW)
move(N)
move(S)
move(S)
move(NW)
move(W)
move(W)
land
liftoff
move(NW)
move(NW)
move(NE)
liftoff
move(E)
move(S)
move(SW)
pass
move(NW)
land
move(E)
move(W)
move(NW)
move(SW)
land
test(25,1,39,46,31)
liftoff
abort
move(SE)
move(NE)
move(SW)
move(N)
move(E)
move(SW)
move(SW)
move(S)
move(S)
move(E)
test(24,27,47,46,54)
move(W)
move(NE)
land
move(W)
move(SE)
move(W)
test(12,29,45,20,50)
test(11,2,28,37,38)
move(S)
