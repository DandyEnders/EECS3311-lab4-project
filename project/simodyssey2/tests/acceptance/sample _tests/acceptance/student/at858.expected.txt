  state:0.0, ok
  Welcome! Try test(3,5,7,15,30)
->move(E)
  state:0.1, error
  Negative on that request:no mission in progress.
->pass
  state:0.2, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.3, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.4, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.5, error
  Negative on that request:no mission in progress.
->pass
  state:0.6, error
  Negative on that request:no mission in progress.
->test(21,9,21,20,35)
  state:0.7, error
  Thresholds should be non-decreasing order.
->move(NE)
  state:0.8, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.9, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.10, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.11, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.12, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.13, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.14, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.15, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.16, error
  Negative on that request:no mission in progress.
->land
  state:0.17, error
  Negative on that request:no mission in progress.
->test(15,24,15,28,36)
  state:0.18, error
  Thresholds should be non-decreasing order.
->move(SW)
  state:0.19, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.20, error
  Negative on that request:no mission in progress.
->pass
  state:0.21, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.22, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.23, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.24, error
  Negative on that request:no mission in progress.
->pass
  state:0.25, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.26, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.27, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.28, error
  Negative on that request:no mission in progress.
->land
  state:0.29, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.30, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.31, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.32, error
  Negative on that request:no mission in progress.
->abort
  state:0.33, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.34, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.35, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.36, error
  Negative on that request:no mission in progress.
->wormhole
  state:0.37, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.38, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.39, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.40, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.41, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.42, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.43, error
  Negative on that request:no mission in progress.
->pass
  state:0.44, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.45, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.46, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.47, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.48, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.49, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.50, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.51, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.52, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.53, error
  Negative on that request:no mission in progress.
->land
  state:0.54, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.55, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.56, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.57, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.58, error
  Negative on that request:no mission in progress.
->liftoff
  state:0.59, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.60, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.61, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.62, error
  Negative on that request:no mission in progress.
->pass
  state:0.63, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.64, error
  Negative on that request:no mission in progress.
->land
  state:0.65, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.66, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.67, error
  Negative on that request:no mission in progress.
->move(NW)
  state:0.68, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.69, error
  Negative on that request:no mission in progress.
->land
  state:0.70, error
  Negative on that request:no mission in progress.
->test(25,1,39,46,31)
  state:0.71, error
  Thresholds should be non-decreasing order.
->liftoff
  state:0.72, error
  Negative on that request:no mission in progress.
->abort
  state:0.73, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.74, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.75, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.76, error
  Negative on that request:no mission in progress.
->move(N)
  state:0.77, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.78, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.79, error
  Negative on that request:no mission in progress.
->move(SW)
  state:0.80, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.81, error
  Negative on that request:no mission in progress.
->move(S)
  state:0.82, error
  Negative on that request:no mission in progress.
->move(E)
  state:0.83, error
  Negative on that request:no mission in progress.
->test(24,27,47,46,54)
  state:0.84, error
  Thresholds should be non-decreasing order.
->move(W)
  state:0.85, error
  Negative on that request:no mission in progress.
->move(NE)
  state:0.86, error
  Negative on that request:no mission in progress.
->land
  state:0.87, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.88, error
  Negative on that request:no mission in progress.
->move(SE)
  state:0.89, error
  Negative on that request:no mission in progress.
->move(W)
  state:0.90, error
  Negative on that request:no mission in progress.
->test(12,29,45,20,50)
  state:0.91, error
  Thresholds should be non-decreasing order.
->test(11,2,28,37,38)
  state:0.92, error
  Thresholds should be non-decreasing order.
->move(S)
  state:0.93, error
  Negative on that request:no mission in progress.
